sdk/eventgrid/Azure.Messaging.EventGrid/README.md

```diff
@@ -187,18 +187,18 @@ foreach (CloudEvent cloudEvent in cloudEvents)
             break;
         case "Microsoft.Storage.BlobDeleted":
             // Example for deserializing system events using GetData<T>
-            StorageBlobDeletedEventData blobDeleted = cloudEvent.GetData<StorageBlobDeletedEventData>();
+            StorageBlobDeletedEventData blobDeleted = cloudEvent.GetData<StorageBlobDeletedEventData>(http://github.com/Azure/azure-sdk-for-net/blob/master/sdk/eventgrid/Azure.Messaging.EventGrid/README.md);
```

Comment:
This seems very wrong

Reply:
Yes this is definitely not correct. @sima-zhu looks like a find/replace `()` is doing the wrong thing.

Reply:
Fixed it.